tests/__snapshots__/settings.test.ts.snap

```diff
@@ -14,6 +14,7 @@ exports[`Test on settings Admin key: Get default settings of an index 1`] = `
     },
   },
   "filterableAttributes": [],
+  "localizedAttributes": null,
```

Comment:
If you rebase on the v1.10 branch, you should retrieve those changes without having to add them in this PR :)